routes/index.js

```diff
@@ -204,6 +207,36 @@ router.delete(
   db.deleteReport
 );
 
+/**
+ * @swagger
+ * /reportdb/report/{report_id}/records:
+ *   get:
+ *     tags:
+ *       - report_record
+ *     description: Returns all report records for a given report ID
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: report_id
+ *         description: report id
+ *         in: path
+ *         required: true
+ *         types: integer
+ *     responses:
+ *       200:
+ *         description: All report records for given report_id
+ *         schema:
+ *           type: array
+ *           items:
+ *             $ref: '#/definitions/report_record_complete'
```

Comment:
Should probably return 404 if the report doesn't exist

Reply:
Good call, will add that

Reply:
Added